playbooks/common/openshift-master/config.yml

```diff
@@ -349,7 +349,8 @@
   roles:
   - role: openshift_master_cluster
     when: openshift_master_ha | bool and openshift.master.cluster_method == "pacemaker"
-  - openshift_examples
+  - role: openshift_examples
+    when: openshift.common.install_openshift_examples | bool
```

Comment:
I don't see where the value provided on the inventory is persisted.  To persist the user provided value, roles/openshift_common/tasks/main.yml will need to be updated to set the value through the local_facts parameter to openshift_facts.